release-notes/release-notes/5.7-release-notes.md

```diff
@@ -314,3 +314,7 @@ We've fixed a bug where users were prompted to restore unsaved changes for a new
 **CIN-04628 If System User is a part of the Connections (or any) group application fails to initialize**
 
 We've fixed a bug that was causing the platform to fail upon initializing when a System User had been added to any user group (such as the Connections or Admin groups).
+
+**CIN-04346 (PATCH) Ensure variable resolution works with encrypted and unencrypted sync configs**
+
+We've fixed a bug where if you passed a parameter unencrypted values during a data sync, it would throw an error that would cause the sync to fail. These parameters now work with either encrypted or plaintext values. 
```

Comment:
I believe the issue actually stemmed from users passing in an encrypted value.

So maybe "We've fixed a bug where if you passed a parameter **encrypted** values during a data sync."

Reply:
@alvin-h fixed. Let me know if it looks good.